### DISCUSSION TITLE
Process nested jar

### DISCUSSION THREAD

**Discussion by w58296** [soot-oss/SootUp#713] [General]
is there any way to process a nested jar file，it seems like sootup skip process second level jar

**Comment by JonasKlauke** [comment]
If I recall correctly there is currently no inputlocation dealing with nested jars.
I had a similar problem with analysing spring boot. I solved it by writing a small programm that creates a FatJar containing everything, because there are also some other issues with Spring boots jar structure. 
An other solution would be to put every nested jar as inputlocation into the view

**Reply by swissiety** [reply]
we should support that.

**Reply by JonasKlauke** [reply]
I wrote it down as an issue  #750
